rotimi-best/classroomio
# feat: verify users email

## What does this PR do?

This PR is a feature to verify users email



Fixes #208 



## Type of change




## How should this be tested?



- Login or Signup
- You should see a Verify popup
- Click it
- You should get an email in your inbox to verify you
- Once you click it, you should be redirected to the dashboard with the welcome modal

## Checklist



### Required


### Appreciated
